README.md

```diff
@@ -13,6 +13,19 @@ poetry install
 poetry shell
 ```
 
+## Running on hardware
+
+### Credential management
+
+To run on hardware, each hardware provider offers API tokens that are required to interact with their quantum devices.
+In order to run on these devices, you will need to follow the instructions on the respective pages of the providers and
+obtain API keys from them.
+
+The `.env.example` file illustrates how to specify the API keys once you have acquired them. You will need to create a
+`.env` file in the same directory as `.env.example` and populate the values of these variables accordingly.
```

Comment:
Should add the `.env` file to the `.gitignore` so that it is not accidentally committed with credentials

Reply:
Yes, absolutely. In fact, it was already present in the `.gitignore` that I initially supplied to the project, so we should be all good there.